src/main/java/com/yahoo/sketches/tuple/DirectArrayOfDoublesCompactSketch.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2015, Yahoo! Inc.
+ * Licensed under the terms of the Apache License 2.0. See LICENSE file at the project root for terms.
+ */
+package com.yahoo.sketches.tuple;
+
+/**
```

Comment:
The javadoc for the class should be after the imports and just before the class declaration.